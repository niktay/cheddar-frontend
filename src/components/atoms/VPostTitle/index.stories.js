import { storiesOf } from "@storybook/vue";
import VPostTitle from ".";

storiesOf("Atom - VPostTitle", module).add("default", () => ({
  components: { VPostTitle },

  template: `
  <VPostTitle>Sample Post Title</VPostTitle>`
}));
