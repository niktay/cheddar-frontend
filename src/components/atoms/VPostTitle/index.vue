<template>
  <h2 class="a-VPostTitle">
    <slot></slot>
  </h2>
</template>

<script>
export default {};
</script>

<style class="scss">
.a-VPostTitle {
  color: #111111;
  font-family: "Merriweather", serif;
  font-size: 2rem;
  font-weight: 900;
  margin-bottom: 7px;
}
</style>
