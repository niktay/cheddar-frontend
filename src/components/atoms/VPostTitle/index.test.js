import { mount } from "@vue/test-utils";
import VPostTitle from ".";

describe("VPostTitle component", () => {
  it("should render correctly", () => {
    const comp = mount(VPostTitle, {
      slots: {
        default: "Post Title"
      }
    });

    expect(comp.html()).toMatchSnapshot();
  });
});
