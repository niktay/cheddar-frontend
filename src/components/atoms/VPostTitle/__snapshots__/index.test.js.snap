// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VPostTitle component should render correctly 1`] = `<h2 class="a-VPostTitle">Post Title</h2>`;
